### PR TITLE
First Boot Wizar in LimeApp

### PR DESCRIPTION
**Pull request by gmarcos87**
**ATTENTION: This is a pull request in process, do not merge until all items are resolved.**

With this pull request is incorporated to lime-app the ability to handle the First Boot Wizard of Libremesh. This will allow the rapid expansion of mesh networks.
It still has to be resolved:
- [x] Improve GUI design
- [x] Allow to create a new network
- [x] In case you don't find any network offer create a new one
- [ ] Add in settings the option to display the FBW screen again
- [ ] Merge this pull request ([FirstBootWizard](https://github.com/libremesh/FirstBootWizard/pull/9))
- [ ] Merge this pull request ([lime-packages-ui](https://github.com/libremesh/lime-packages-ui/pull/21))
- [ ] Add FirstBootWizard to lime-packages
- [x] Cook the Libremesh image with FBW and test the concept multiple times.

### CODE REVIEW

File: plugins/lime-plugin-fbw/src/actions.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+};
+
+export const setNetwork = ( file ) => ( dispatch ) => {
+	dispatch({
+		type: FBW_SET_NETWORK,
+		payload: file
+	});
+};
+
+export const createNetwork = (network) => (dispatch) => {
```
Comment (crnd) [inline plugins/lime-plugin-fbw/src/actions.js]: Is there a reason why some parentheses have spaces inside like ```( file )``` but here ```(network)``` is without?

File: plugins/lime-plugin-fbw/src/page.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+class Page extends Component {
+
+	searchNetworks() {
+		this.loop = setInterval(() => {
+			if (this.props.status === 'scanned'){
+				clearInterval(this.loop);
+				return;
+			}
+			this.props.searchNetworks(this.props.status === 'scanned');
```
Comment (crnd) [inline plugins/lime-plugin-fbw/src/page.js]: Isn't the value of ```this.props.status === 'scanned'``` already known here?

File: plugins/lime-plugin-fbw/src/page.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+		this.setNetwork = this.setNetwork.bind(this);
+	}
+
+	componentWillUnmount(){
+		if (this.loop) {
+			clearInterval(this.loop);
+		}
+	}
+
+	render (){
```
Comment (crnd) [inline plugins/lime-plugin-fbw/src/page.js]: Inconsistencies also here with spacing: ```setNetwork() {```, ```componentWillUnmount(){``` and ```render (){```

File: plugins/lime-plugin-fbw/src/actions.js (hunk 1 above)
Comment (gmarcos87) [reply plugins/lime-plugin-fbw/src/actions.js]: Hi crnd, are you a Bot?

Comment (crnd) [reply plugins/lime-plugin-fbw/src/actions.js]: Not to my knowledge.

Comment (gmarcos87) [reply plugins/lime-plugin-fbw/src/actions.js]: Okay, thanks for your input. That's where I applied the changes. It's rare to see someone here looking at the code. Are you a user of libremesh? Are you interested in contributing or coordinating some revisions?

Comment (crnd) [reply plugins/lime-plugin-fbw/src/actions.js]: I was just browsing through and noticed that there are plenty of inconsistencies in the code style.

**Pull request merged**